Content/build.fsx

```diff
@@ -137,17 +185,46 @@ Target.create "Bundle" (fun _ ->
 
 let dockerUser = "safe-template"
 let dockerImageName = "safe-template"
+//#endif
+//#if (deploy == "docker")
 let dockerFullName = sprintf "%s/%s" dockerUser dockerImageName
 
 Target.create "Docker" (fun _ ->
-    let buildArgs = sprintf "build -t %s ." dockerFullName
-    runTool "docker" buildArgs "."
+    buildDocker dockerFullName
+)
+
+//#endif
+//#if (deploy == "gcp-kubernetes")
+Target.create "Docker" (fun _ ->
+    let dockerTag = createDockerTag dockerImageName
+    buildDocker dockerTag
+)
+
+Target.create "Publish" (fun _ ->
+    let dockerTag = createDockerTag dockerImageName
+    let pushArgs = sprintf "push %s" dockerTag
+    runTool "docker" pushArgs "."
```

Comment:
`unauthorized: You don't have the needed permissions to perform this operation, and you may have invalid credentials. To authenticate your request, follow the steps in: https://cloud.google.com/container-registry/docs/advanced-authentication` <- should be probably reflected in docs?